### PR TITLE
[doc] Update Contributing Guide

### PR DESCRIPTION
**Pull request by roubkar**
- Separated COND_OF_CONDUCT
- Added Contributing Guide

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,92 +1,155 @@
...
-Examples of behavior that contributes to creating a positive environment
-include:
+#### Authors
 
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
+- Gevorg Soghomonyan ([SGevorg](https://github.com/SGevorg))
```
Comment (SGevorg) [inline CONTRIBUTING.md:24]: Gev Sogomonian

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+### Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (SGevorg) [inline CODE_OF_CONDUCT.md:58]: [community@aimstack.io](mailto:community@aimstack.io)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,92 +1,155 @@
...
 
-* The use of sexualized language or imagery and unwelcome sexual attention or
-advances
-* Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
-* Publishing others' private information, such as a physical or electronic
-  address, without explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
-  professional setting
+- [alberttorosyan](https://github.com/alberttorosyan)
```
Comment (SGevorg) [inline CONTRIBUTING.md:29]: I suggest the following structure:
**Authors**
Gor, Gev

**Primary maintainers**
Ruben K, Albert, mahnerak

**Core Maintainers**
Aram
Arsen
Arthur
Hamik
Karo
Mihran
Ruben A.
Vardges

**Review by SGevorg — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (SGevorg) [reply CONTRIBUTING.md:29]: Core contributors vs Core maintainers @gorarakelyan thoughts?

Comment (gorarakelyan) [reply CONTRIBUTING.md:29]: sounds good

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -21,21 +21,24 @@ This section describes governance processes we follow in developing Aim.
...
+#### Primary Maintainers
 
 - [alberttorosyan](https://github.com/alberttorosyan)
+- Karen Hambardzumyan ([mahnerak](https://github.com/mahnerak))
+- Ruben Karapetyan ([roubkar](https://github.com/roubkar))
+
+#### Core Maintainers
+
 - [aramaim](https://github.com/aramaim)
 - Arsen Alkamyan ([arsengit](https://github.com/arsengit))
```
Comment (alberttorosyan) [inline CONTRIBUTING.md:35]: would be nice to keep the same style here:
FirsName LastName (GH Account link)

Comment (roubkar) [reply CONTRIBUTING.md:35]: @alberttorosyan latest changes include that fix

**Review by SGevorg — APPROVED** [review]
lgtm

**Pull request merged**